src/execute.js

```diff
@@ -5,17 +5,19 @@ const log = require('./log');
 
 module.exports = function execute(config) {
   const gitOrigin = config.useRepo ?
-    buildGitUrl(config.repoOrg, config.repoName) : false;
+    buildGitUrl(config.repoOrg, config.repoName, config.useHttps) : false;
   const gitFork = config.repoForkOrg ?
-    buildGitUrl(config.repoForkOrg, config.repoName) : false;
+    buildGitUrl(config.repoForkOrg, config.repoName, config.useHttps) : false;
 
   clone(config.techStack, gitFork, gitOrigin, config.repoName)
     .catch(log);
 };
 
-function buildGitUrl(orgName, projectName) {
-  return `git@github.org:${orgName}/${projectName}.git`;
-}
+const HTTPS = 'https://github.com/';
```

Comment:
Maybe another choice in inquirer for the SSH/HTTPS section - could even be "other: (enter url)" or something.  But we can do that later.